mne_bids/stats.py

```diff
@@ -0,0 +1,110 @@
+"""Some functions to extract stats from a BIDS dataset."""
+
+# Authors: Alex Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+
+from mne_bids import BIDSPath, get_datatypes
+from mne_bids.config import EPHY_ALLOWED_DATATYPES
+
+
+def count_events(root_or_path, datatype='auto'):
+    """Count events present in dataset.
+
+    Parameters
+    ----------
+    root_or_path : str | pathlib.Path | BIDSPath
+        If str or Path it is the root folder of the BIDS dataset.
+        If a BIDSPath is passed it allows to limit the count
+        to a subject, a session or a run by only considering
+        the event files that match this BIDSPath.
+    datatype : str
+        Type of the data recording. Can be ``meg``, ``eeg``,
+        ``ieeg`` or ``auto``. If ``auto`` then either a BIDSPath
+        isinstance is passed as ``root_or_path`` and this is what is
+        used or otherwise only one datatype should be present in
+        the dataset to avoid any ambiguity.
+
+    Returns
+    -------
+    counts : pd.DataFrame
+        The pandas dataframe containing all the counts of trial_type
+        in all matching events.tsv files.
+    """
+    import pandas as pd
+
+    if not isinstance(root_or_path, BIDSPath):
+        bids_path = BIDSPath(root=root_or_path)
+    else:
+        bids_path = root_or_path.copy()
+
+    bids_path.update(suffix='events', extension='tsv')
+
+    datatypes = get_datatypes(bids_path.root)
+    this_datatypes = list(set(datatypes).intersection(EPHY_ALLOWED_DATATYPES))
+
+    if (datatype == 'auto') and bids_path.datatype is not None:
```

Comment:
```suggestion
    if datatype == 'auto' and bids_path.datatype is not None:
```

Reply:
I rather added some () :)

Reply:
Yes I like the parens ;)

Reply:
By why not put parens around the second term too :)

Reply:
I won't question French aesthetics :)